moabb/paradigms/base.py

```diff
@@ -265,6 +267,13 @@ def get_data(  # noqa: C901
             This pipeline must return an ``np.ndarray``.
             This pipeline must be "fixed" because it will not be trained,
             i.e. no call to ``fit`` will be made.
+        additional_metadata: Literal["default", "all"] | list[str]
+            Additional metadata to be loaded if return_epochs=True.
```

Comment:
The `get_data()` function returns a triplet `(obj, labels, metadata)`. 
`obj` contains the data and can be a `np.array`, `mne.Epochs` or `mne.io.Raw` depending on the `return_epochs` and `return_raws` parameters.
But we should always return some metadata, so the additional columns should always be set when `additional_metadata='all'`